source/frontend/assets/scripts/database.js

```diff
@@ -22,17 +22,17 @@ async function loadChallenges() {
  * Fetches the challenge list from file.
  * @returns {Promise} Resolves with the challenge list json if successful, rejects otherwise.
  */
-async function loadChallengesFromFile() {
-  return new Promise((resolve, reject) => {
-    fetch('assets/jsons/challenges.json')
-      .then((response) => response.json())
-      .then((data) => {
-        resolve(data);
-      })
-      .catch((error) => {
-        reject(error);
-      });
-  });
+async function loadChallengesFromServer() {
+  try {
+    const response = await fetch(`${url}/recipes/challenges`);
+    const data = await response.json();
+    const result = {
+      challenges: data.challengesWithRecipes,
+    };
+    return result;
+  } catch (err) {
+    return err;
+  }
 }
```

Comment:
I tested the router which worked as expected, but how do test this function? I tried going to `http://exploding-kitchen.us-west-1.elasticbeanstalk.com/api` but it won't load the page. Am I running the app incorrectly? I ran nodemon inside of the backend folder

Reply:
The link you are referring to won't work until this is merged and deployed. The link is the production link.